product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/src/main/synapse-config/proxy-services/1_6_1_2_SoapToPoxEnrichWithXpathAsChildProxy.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<proxy name="1_6_1_2_SoapToPoxEnrichWithXpathAsChildProxy" startOnLoad="true" transports="http https" xmlns="http://ws.apache.org/ns/synapse">
+    <target>
+        <inSequence>
+            <log>
```

Comment:
log level defaults to 'simple' and the headers 'To, From, WSAction, SOAPAction, ReplyTo, MessageID' are logged in that level. Do we need those headers to get printed? If not, shall we set the log level to 'custom' in all application places?

Reply:
I think it does not add a high overhead. Also, these information can be important if something went wrong. Synapse designers have kept it default considering that fact I believe. I vote we will keep it as it is.